OBAKitCore/Bookmarks/TripBookmarkKey.swift

```diff
@@ -28,7 +28,8 @@ public struct TripBookmarkKey: Hashable, Equatable {
     }
 
     public init(arrivalDeparture: ArrivalDeparture) {
```

Comment:
it seems confusing to me to keep this `init` method around—only to have it generate a `fatalError()` if called. Seems like a major footgun. Please delete unless I'm missing something important.

Reply:
I wrote these fatalErrors in areas that used to rely on the `loadReferences` infrastructure, but I don't yet fully understand how OBAKit uses them. It exists to temporarily satisfy the compiler until I go back in a later PR to address them. I updated the error string to include `"PR-686 error"` to grep later.